hexrdgui/masking/mask_manager.py

```diff
@@ -0,0 +1,390 @@
+import math
+import numpy as np
+
+from PySide6.QtCore import Signal, QObject
+from hexrdgui import utils
+
+from hexrdgui.constants import ViewType
+from hexrdgui.masking.constants import MaskType
+from hexrdgui.masking.create_polar_mask import (
+    create_polar_mask_from_raw, rebuild_polar_masks
+)
+from hexrdgui.masking.create_raw_mask import (
+    recompute_raw_threshold_mask, create_raw_mask, rebuild_raw_masks
+)
+from hexrdgui.hexrd_config import HexrdConfig
+from hexrdgui.masking.mask_compatability import load_masks_v1_to_v2
```

Comment:
Spelling correction:

```suggestion
from hexrdgui.masking.mask_compatibility import load_masks_v1_to_v2
```